### PR TITLE
Add accessibility guidance for Placeholders

### PR DESCRIPTION
**Pull request by ericwbailey**
This PR adds guidance in the Accessibility section for placeholder content. It is intended to serve a more thorough reference than the [text input accessibility subsection](https://primer.style/components/text-input#accessibility), for folks who would like to learn more about some of the issues.

### CODE REVIEW

File: content/guides/accessibility/placeholders.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- [Avoid using placeholders](https://primer.style/components/text-input#accessibility) if at all possible.
+- Instead, use a [caption](https://primer.style/ui-patterns/forms/overview#caption) to communicate this information.
+- If you must use a placeholder, be sure that the input also has a concise, descriptive label.
+
+## Why?
+
+Placeholders are typically styled using a light text treatment to visually communicate that they are suggestions, and not pre-filled input. However, this often means they are too light to be color contrast compliant. If the placeholder color treatment is darked to be compliant, it risks being visually interpreted as pre-filled input.
+
+Placeholder length is also contrained by the width of the input it is used in. This means important information may be truncated, especially if the user relies on an increased font size.
```
Comment (ichelsea) [inline content/guides/accessibility/placeholders.mdx:20]:
```suggestion
Placeholder length is also constrained by the width of the input it is used in. This means important information may be truncated, especially if the user relies on an increased font size.
```

File: content/guides/accessibility/placeholders.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: Placeholders
+description: Placeholders have many accessibility and usability concerns to be aware of.
+---
+
+## Overview
+
+Placeholders are hint text placed inside of an input. They are used to help suggest waht kind of input the should be provided, but have some accessibility considerations to be aware of.
```
Comment (langermank) [inline content/guides/accessibility/placeholders.mdx:8]:
```suggestion
Placeholders are hint text placed inside of an input. They are used to help suggest what kind of input should be provided, but have some accessibility considerations to be aware of.
```

**Review by langermank — APPROVED** [review]
Just a small typo fix suggestion

File: content/guides/accessibility/placeholders.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### Best practices
+
+- [Avoid using placeholders](https://primer.style/components/text-input#accessibility) if at all possible.
+- Instead, use a [caption](https://primer.style/ui-patterns/forms/overview#caption) to communicate this information.
+- If you must use a placeholder, be sure that the input also has a concise, descriptive label.
+
+## Why?
+
+Placeholders are typically styled using a light text treatment to visually communicate that they are suggestions, and not pre-filled input. However, this often means they are too light to be color contrast compliant. If the placeholder color treatment is darked to be compliant, it risks being visually interpreted as pre-filled input.
```
Comment (ericwbailey) [inline content/guides/accessibility/placeholders.mdx:18]:
```suggestion
Placeholders are typically styled using a light text treatment to visually communicate that they are suggestions, and not pre-filled input. However, this often means they are too light to be color contrast compliant. If the placeholder color treatment is darkened to be compliant, it risks being visually interpreted as pre-filled input.
```

**Pull request merged**